src/main/java/core/basesyntax/RightTriangleSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RightTriangleSupplier {
+    private Random random = new Random();
+    private RandomSize size = new RandomSize();
+    private ColorSupplier color = new ColorSupplier();
+
+    public Figure getRandomRightTriangle() {
+        return new RightTriangle(size.getRandomSize(),

```

Comment:
RightTriangle is being created with same length for both legs, this will not create random right triangles. Two different random values should be used for the lengths of the legs.

Reply:
I guess they're not the same:
right triangle, area: 2.89115424E8 sq. units, first side: 39432 units, second side: 14664 units`